### PR TITLE
Fixed grammar mistakes

### PR DESCRIPTION
**Pull request by VerisimilitudeX**

### CODE REVIEW

File: src/routes/+page.svelte [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <h1>Nv7's Projects</h1>
 <p class="lead">
-  Welcome to Nv7's Projects! This is where Nv7 keeps all of his fun or interesting programs! This page is being rendered via <a href="https://kit.svelte.dev/">SvelteKit</a> and written in <a href="https://www.typescriptlang.org/">TypeScript</a>. Thanks for check out Nv7's Projects and be sure to try some of the projects!
+  Welcome to Nv7's Projects! This is where Nv7 keeps all of his fun and interesting programs! This page is being rendered via <a href="https://kit.svelte.dev/">SvelteKit</a> and is written in <a href="https://www.typescriptlang.org/">TypeScript</a>. Thanks for checking out Nv7's Projects and make sure to try some of the projects!
```
Comment (Nv7-GitHub) [inline src/routes/+page.svelte]: I meant it to be `fun or interesting` because some of them are fun and some of them are interesting

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]
One thing

File: src/routes/+page.svelte (hunk 1 above)
Comment (VerisimilitudeX) [reply src/routes/+page.svelte]: Alright

**Review by Nv7-GitHub — APPROVED** [review]
Looks good!

**Pull request merged**